.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+version: 2.1
+jobs:
+  build:
+    docker:
+      - image: docker:18.06.3-ce-git
+    steps:
+      - checkout
+      - setup_remote_docker
+      - restore_cache:
+          keys:
+            - v1-{{ .Branch }}
+          paths:
+            - /caches/yoba.tar
+      - run:
+          name: Load Docker image layer cache
+          command: |
+            set +o pipefail
+            docker load -i /caches/yoba.tar | true
```

Comment:
Is this error handling? :)

Reply:
I suppose it is, I just copy-pasted it from docs